src/components/DirectoryRootGrid.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from "react";
+import { twMerge } from "tailwind-merge";
+import { formatPhoneNumber, formatPhoneNumberIntl } from 'react-phone-number-input';
+
+
+export const directoryRootGridFields = [
```

Comment:
I think this should be removed. See my comment in location.tsx

Reply:
Fair, though worth noting I copied this from Consulting's repo. I assume your reasoning is just that you like having everything in one spot? which, I am in agreement with